dubbo-sample-gateway/src/main/java/org/apache/dubbo/samples/CallbackListenerImpl.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.dubbo.samples;
+
+
+import org.apache.dubbo.samples.callback.api.CallbackListener;
+
+/**
+ * @author complone
```

Comment:
pls. remove author info, use English for comment, and add apache license header.